packages/client/src/components/ModalComponent/ModalComponent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './modal.styles.css';
+
+const ModalComponent = ({ handleModal }) => {
+  return (
+    <div className="overlay">
+      <div className="modal-container">
+        <h5>This is the review pop up</h5>
```

Comment:
This modal is not just for reviews, so let's start with dynamic body content right away 🙏🏽

Reply:
I think it's done now